workflows/proteinannotator.nf

```diff
@@ -24,6 +25,10 @@ workflow PROTEINANNOTATOR {
     ch_versions = Channel.empty()
     ch_multiqc_files = Channel.empty()
 
+    // todo: move this to stats on input fasta subworkflow
+    SEQKIT_STATS(ch_samplesheet)
+    ch_multiqc_files = ch_multiqc_files.mix(SEQKIT_STATS.out.versions)
+
```

Comment:
As the comment mentions, this should be moved to a separate stats on input fasta subworkflow later